packages/valory/connections/p2p_libp2p/tests/test_aea_cli.py

```diff
@@ -65,7 +65,6 @@ def setup_class(cls) -> None:
     @classmethod
     def teardown_class(cls) -> None:
         """Tear down the test"""
-        cls.terminate_agents()
```

Comment:
this leads to errors this method should be augmented to include additional logic needed seen in `teardown_class`. It may also be cause by conditionals in nested calls of the `teardown_class` itself, I did not check yet. Alternatively we may consider making this a private method.